### DISCUSSION TITLE
hive元数据相关

### DISCUSSION THREAD

**Discussion by tiger-hcx** [apache/doris#16000] [A - General / Q&A]
doris的版本是1.2.1.
场景：hudi数据写入的时候并同步元数据到hive，然后在doris中创建catalog，然后查询hudi的数据。
创建catalog：
CREATE CATALOG hive7 PROPERTIES (
    'type'='hms',
    'hive.metastore.uris' = 'thrift://ip:9083'
);
问题是：创建完catalog之后可以正常查询现有hudi的数据，但是新建一张hudi表的时候，新表元数据已经同步到hive，使用hive也可以正常查询hudi数据，但是使用doris查不到新建的这张表，需要重新新建一个catalog才能查到这张表。也就是doris连接hive元数据并不是实时同步的，只是同步了创建catalog的时候的hive元数据，这个问题如何解决呢？

**Comment by zy-kkk** [comment]
可以通过命令 `REFRESH CATALOG catalog_name;` 来刷新元数据
后续1.2.2会支持自动刷新
